Language/Variables/Implicitly_Induced_Getters_and_Setters/final_A02_t01.dart

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2011, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion A non-local variable introduces a setter iff it does not have the
+/// modifier final or the modifier const, or it is late and final, but does not
+/// have an initializing expression.
```

Comment:
This assertion doesn't fit the test: The description and the test is about a local variable, but the assertion is about all those kinds of variables that aren't local.

It's a somewhat inconvenient situation: I don't know exactly how the assertions are organized, but if a test needs to have its assertion updated then I wouldn't be surprised if it would then also need to be renamed and put into a different directory.

We do want to have the test which is actually written here. There is of course another case where the variable is final, isn't initialized in the declaration, and is evaluated at a point where it isn't guaranteed to have been initialized, or it is assigned at a time where it is definitely already initialized, but I think we have those elsewhere already.

Maybe this test is essentially a duplicate, and it should simply be deleted?

Reply:
Updated assertion and moved this and similar tests to `Srtatements/Local Variable Declaration` directory

Reply:
Sounds good!